contracts/src/NativeTokenDestination.sol

```diff
@@ -401,6 +401,7 @@ contract NativeTokenDestination is
      */
     function _mintNativeCoin(address recipient, uint256 amount) private {
         totalMinted += amount;
+        emit NativeCoinsMinted(recipient, amount);
```

Comment:
We originally removed this because the native minter precompile emits an almost identical event. I don't really have a strong opinion though

Reply:
I also don't have a strong opinion here, since as you stated, the native minter precompile has pretty much same event. My only thinking was in case of other `INativeTokenDestination` implementations that for some reason don't use the native minter precompile.